include/kernels/clover_sigma_outer_product.cuh

```diff
@@ -21,13 +21,18 @@ namespace quda
     using F = typename colorspinor_mapper<Float, nSpin, nColor>::type;
 
     Oprod oprod;
+    const bool doublet;         // whether we applying the operator to a doublet
+    const size_t VolumeCB;
```

Comment:
I don't think there's any need to make this a `size_t`, and `unsigned int` or `int` is sufficient.  Also could define it to include the factor of half to avoid needed to divide it in the kernel, e.g.
```
volume_4d_cb(inA[0].VolumeCB() / 2)
```

Reply:
[57a6e9b](https://github.com/lattice/quda/pull/1452/commits/57a6e9bc214734096b84061971bac8dd77bfd84c)